runtime/kusama/src/lib.rs

```diff
@@ -935,6 +938,13 @@ impl frame_support::traits::OnRuntimeUpgrade for CustomOnRuntimeUpgrade {
 	}
 }
 
+pub struct PhragmenElectionDepositRuntimeUpgrade;
+impl frame_support::traits::OnRuntimeUpgrade for PhragmenElectionDepositRuntimeUpgrade {
+	fn on_runtime_upgrade() -> frame_support::weights::Weight {
+		pallet_elections_phragmen::migrations::migrate_to_recorded_deposit::<Runtime>(5 * DOLLARS)
```

Comment:
```suggestion
		pallet_elections_phragmen::migrations::migrate_to_recorded_deposit::<Runtime>(5 * CENTS)
```

Err this must actually be the old voting bond.. please double check and apply upon approve.